### PR TITLE
bump defaultReconciliationTimeout to 60 sec

### PR DESCRIPTION
**Pull request by andrewhsu**
**- What I did**

bump `defaultReconciliationTimeout` from 30 sec to 60 sec

To accommodate for [increase in leader election from 3 seconds to 10
seconds](https://github.com/moby/moby/pull/36701). This will give more cycles to find another leader.

**- How I did it**

in the tests

**- How to verify it**

run `DockerSwarmSuite.TestAPISwarmLeaderElection` test

**- Description for the changelog**

N/A

**- A picture of a cute animal (not mandatory but encouraged)**
🐋

### CODE REVIEW

**Comment by andrewhsu** [comment]
cc @cyli

**Review by cyli — APPROVED** [review]
LGTM - I'm not sure if this bumps the tests up enough - it was 30 seconds before, which means that if the leader election tick were 3 seconds, it's possible to get up to 10 leader elections.  With 60 seconds and a 10 second election tick, it's possible to get up to 6.

I think this will be sufficient, but just wanted to mention that it doesn't give us the exact number of possible elections we might have had before.

File: integration-cli/docker_api_swarm_test.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 	"golang.org/x/net/context"
 )
 
-var defaultReconciliationTimeout = 30 * time.Second
+var defaultReconciliationTimeout = 60 * time.Second
```
Comment (anshulpundir) [inline integration-cli/docker_api_swarm_test.go]: Please add a comment for how this timeout is used.

**Review by anshulpundir — APPROVED** [review]

**Comment by andrewhsu** [comment]
@anshulpundir for example: https://github.com/docker/docker-ce/pull/492#issuecomment-378758146

**Comment by tonistiigi** [comment]
If this is for a specific test then there is no need to raise it for everything. We already raise it in couple of places where needed, eg. https://github.com/moby/moby/blob/a15cbcb3cfc320430c3af027c3e384466cd4c1cf/integration-cli/docker_api_swarm_node_test.go#L113

**Comment by anshulpundir** [comment]
I agree with @tonistiigi's suggestion to only raise it for the test in question. @andrewhsu

**Comment by cyli** [comment]
I am fine with bumping it up only for this test - but there are a lot of other tests that do not specifically test leader election, but may trigger leader elections (because we are taking down nodes), which could become more flaky as well.

**Comment by anshulpundir** [comment]
Agree with @cyli. We can always change increase the timeout for tests as and when necessary.

BTW, there are concerns that we might be masking bugs. And so maybe try with a smaller timeout increase ?

**Pull request closed without merging**